app/src/theme/default-theme.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import ITheme from '../definitions/theme-interface'
+
+export const defaultTheme: ITheme = {
```

Comment:
Wondering why the interface and this default theme is not maintained by leo. Is it particular to swap project?

Reply:
Is not maintained yet as far as I can tell. We can easily switch out in the future though!